events/2024.11.Munich/VPN_static_addr.md

```diff
@@ -0,0 +1,15 @@
+# IP address allocation table for Plugfest VPN
+
+In plugfest VPN network, the DHCP server dynamically allocates IP addresses between 192.168.30.128 to 250.  
+If you would like to use statically allocated IP addresses, please secure your addresses
+by updating this markdown file and making a pull request.
+
+| IP Address | Description |
+| --- | --- |
+|192.168.30.1       | Gateway         |
+|192.168.30.2-9     | Reserved        |
```

Comment:
Could explain who is it reserved for?

Reply:
These are reserved in case it become necessary to connect machines for the management of the VPN itself.
So, how about updating "Reserved for administrative purpose"?

Reply:
Ah I see ok. Yes it that reformulation would be clearer :)